docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-navigated-viewer.production.min.js"></script>
+```
+
+### Including via NPM
+
+First, install bpmn-js via NPM:
+```
+npm install bpmn-js
+```
+
+Then import the BPMN viewer via an ES:
+
+```javascript
+import Viewer from 'bpmn-js/lib/Viewer';
+```
+
+or for the navigated viewer:
+
+```javascript
+import NavigatedViewer from 'bpmn-js/lib/NavigatedViewer';
+```
+
+### Viewing the BPMN 2.0 XML
+
+Now that we're done with the setup, we should construct a new bpmn-js Viewer instance. If we imported the bpmn-js as pre-packaged, we should use the `BpmnJS` constructor:
+
+```javascript
+const viewer = new BpmnJS({ container: '#canvas' });
+```
+
+if we've imported the bpmn-js via NPM, we should go for the `Viewer` constructor instead:
+```javascript
+const viewer = new Viewer({ container: '#canvas' });
+```
+
+or for the navigated viewer:
+```javascript
+const viewer = new NavigatedViewer({ container: '#canvas' });
+```
+
+Note that `#canvas` here refers to the [root DOM element that we've created in our HTML file](#including-the-root-element).
+
+We can then import a BPMN 2.0 XML using the [importXML API](/general/api-reference/bpmn-js/importXML). For this demo, we can use this simple BPMN file:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_0xcv3nk" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="3.0.0">
+  <bpmn:process id="Process_0sckl64" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1" />
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_0sckl64">
+      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
+        <dc:Bounds x="179" y="159" width="36" height="36" />
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</bpmn:definitions>
+```
+
+We can directly set this to a variable or we can store this file somewhere and fetch it. For simplicity, we won't deal with this here:
+
+```javascript
+const xml = ' ... ' // We'll assume this variable contains the XML content above.
+
+try {
+
+  await viewer.importXML(xml);
+
+  console.log('XML imported successfully');
+} catch (err) {
+
+  console.log('An error happened importing XML:', err);
+}
+```
+
+### Putting It All Together
+
+See this [Sandbox](https://codesandbox.io/s/quizzical-waterfall-8ffif?file=/src/index.js).
+
+## Using bpmn-js as Modeler
+
+bpmn-js modeler extends the functionality of the [viewer](#using-bpmn-js-as-viewer) by allowing users to interact with the root element and add/remove/edit BPMN elements via a graphical interface. A live example of bpmn-js modeler can be found [here](https://demo.bpmn.io/new).
```

Comment:
what do you mean with the root element? The canvas? Basically, the user can interact with many other stuff, not just with the root element. Maybe

> by allowing users to interact **with the diagram on the canvas** and ...

Reply:
Done 👍